### PR TITLE
Update nebula gradle plugin

### PR DESCRIPTION
**Pull request by stevegury**
Update the gradle-rxjava-project-plugin to the latest version.
Update gradle to version 2.10 as well.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,4 +32,5 @@ if (project.hasProperty('release.useLastTag')) {
 
 test{
      maxHeapSize = "2g"
+     maxParallelForks = 8
```
Comment (akarnokd) [inline build.gradle]: This could increase the likelihood time-sensitive tests fail due to higher server utilization.

**Comment by hzsweers** [comment]
Gradle 2.11 is out, can you not use that?

File: build.gradle (hunk 1 above)
Comment (stevegury) [inline build.gradle]: Indeed, I will remove this.

**Comment by stevegury** [comment]
@hzsweers I didn't check recently, it has just been released!
I've updated the version to 2.11.

**Comment by hzsweers** [comment]
:+1:

**Comment by akarnokd** [comment]
:+1:

**Comment by zsxwing** [comment]
Also cherry-pick to 2.x?

**Comment by stevegury** [comment]
Done -> #3693

**Pull request merged**